resolvers.ts

```diff
@@ -163,97 +171,20 @@ export const resolvers: Resolvers = {
             tissue: input?.todoRemove?.tissue,
             visibility: input?.todoRemove?.visibility,
             workflow: input?.todoRemove?.workflow,
-            // sampleIds and workflowRunIds are only used for API testing, not in the app.
             workflowRunIds: input?.todoRemove?.workflowRunIds,
             sampleIds: input?.todoRemove?.sampleIds,
-
-            //  - DiscoveryDataLayer.ts
-            //    await this._collection.fetchDataCallback({
-            limit: input?.limit,
-            offset: input?.offset,
+            limit: TEN_MILLION,
+            offset: 0,
             listAllIds: false,
           }),
         args,
         context,
       });
-      if (!workflow_runs?.length) {
```

Comment:
@bchu1 removing the big blob below broke API testing :( I was using these fields to compare consensus genomes. Any way we can add them back?